test-data/unit/check-generics.test

```diff
@@ -1017,15 +1017,19 @@ CA = Callable[[T], int]
 TA = Tuple[T, int]
 UA = Union[T, int]
 
-cs = CA[str] + 1 # E: Unsupported left operand type for + ("Type alias to Callable")
+cs = CA + 1 # E: Unsupported left operand type for + ("Type alias to Callable")
```

Comment:
It is less relevant because it will fail with something different like "Type application doesn't target a class" in all three cases. But I will add one more test for this.